app/controllers/admin/applicants_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Admin::ApplicantsController < ApplicationController
+  def index
+    @applicants = Applicant.where(application_status: "Pending")
```

Comment:
So here you are using the `.where` AR method. I think you would probably want to move this to the `applicants` model and make a method that you can call here in your controller.